flatgfa/src/cli/cmds.rs

```diff
@@ -299,3 +301,50 @@ pub fn gaf_lookup(gfa: &flatgfa::FlatGFA, args: GAFLookup) {
         }
     }
 }
+
+///
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "seq-import")]
+pub struct SeqImport {
+    #[argh(positional)]
+    filename: String,
+}
+
+pub fn seq_import(args: SeqImport) {
+    let vec = packedseq::import(&args.filename);
+    let _ = std::fs::remove_file(&args.filename);
+    let store = PackedSeqStore::create(vec);
+    let view = store.as_ref();
+    print!("{}", view);
+}
```

Comment:
It seems like this short function, which is for decompressing a packed sequence file, actually does an extra round trip:

* `import`: Gets the compressed data from disk, and decompresses into an in-memory vector.
* `PackedSeqStore::create`: Compresses uncompressed data into the packed format.
* `PackedSeqView::fmt`: Decompresses the data again so we can format it for the console.

That is, we decompress, recompress, and then decompress again. Would it work to instead just provide a way to get a `PackedSeqView` directly from a file (that's probably what we want, as opposed to what `import` currently does) and then `print!` that?